resources/mocks/point/create_response.json

```
@@ -0,0 +1,15 @@
+{
+  "id": "7f25f9aa-eea6-4f9c-bf16-a341f71ba2f1",
+  "device_id": "PAX_A910__SMARTPOS1234345545",
+  "amount": 1500,
```

Bom pegar numeros quebrados para validar o float
```suggestion
  "amount": 1500.03,
```

---

Amount não é float